mmdeploy/apis/pytorch2onnx.py

```diff
@@ -28,6 +28,13 @@ def torch2onnx_impl(model: torch.nn.Module, input: torch.Tensor,
     backend = get_backend(deploy_cfg).value
     opset_version = pytorch2onnx_cfg.get('opset_version', 11)
 
+    input_names = pytorch2onnx_cfg['input_names']
+    output_names = pytorch2onnx_cfg['output_names']
+    dynamic_axes = pytorch2onnx_cfg.get('dynamic_axes', None)
+
+    if dynamic_axes is not None and not isinstance(dynamic_axes, Dict):
+        dynamic_axes = dict(zip(input_names + output_names, dynamic_axes))
```

Comment:
I think this line is important and our test cases should cover it.

Reply:
Test `test_torch2onnx` has been rewritten to check this case.